src/data_types.ts

```diff
@@ -598,8 +575,23 @@ export class DataTypes extends DataType {
     }
   }
 
-  toSqlString(): string {
-    return '';
+  static get invokable() {
+    return new Proxy(this, {
+      get(target, p) {
+        const value = target[p];
+        if (AllDataTypes.hasOwnProperty(p)) return invokableFunc(value);
+        return value;
+      }
+    }); 
+  }
+
+  /**
+   * Check if params is instance of DataType or not
+   * @param {*} params
+   * @returns {boolean}
+   */
+   static is(params: any): boolean {
+    return params instanceof DataType;
```

Comment:
further refactor the default export `class DataTypes` and the base `class DataType`, which can be imported as `import DataTypes, { DataType } from './data_types';`

existing code that relies on `const { DataTypes } = require('./data_types')` is changed to `const { default: DataTypes } = require('./data_types')` accordingly